10485.xml

```diff
@@ -0,0 +1,191 @@
+			</Item>
+			<Item ID="7">
+				<Name>Full Network Data</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Boolean</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units></Units>
+				<Description><![CDATA[Neighbor mode N represents "Full network data".]]></Description>
+			</Item>
+			<Item ID="8">
+				<Name>Extended MAC Address </Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>String</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units></Units>
+				<Description><![CDATA[ Extended Medium Access Control (MAC).]]></Description>
+			</Item>
+			<Item ID="9">
+				<Name>Link Quality In</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Integer</Type>
+				<RangeEnumeration>Link Quality is an integer in [0, 3]. A higher link quality indicates a more usable link, with 0 indicating that the
+link is non-existent or unusable.</RangeEnumeration>
+				<Units></Units>
+				<Description><![CDATA[Neighbor Link quality In.]]></Description>
+			</Item>
+			<Item ID="10">
+				<Name>Link Quality Out</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Integer</Type>
+				<RangeEnumeration>Link Quality is an integer in [0, 3]. A higher link quality indicates a more usable link, with 0 indicating that the
+link is non-existent or unusable.</RangeEnumeration>
+				<Units></Units>
+				<Description><![CDATA[Neighbor Link quality Out.]]></Description>
+			</Item>		
+				<Item ID="11">
+				<Name>Frame Error</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Float</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units>/100</Units>
+				<Description><![CDATA[Frame error percentage.]]></Description>
+			</Item>	
+				<Item ID="12">
+				<Name>Message Error</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Float</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units>/100</Units>
+				<Description><![CDATA[Message error percentage.]]></Description>
+			</Item>	
+				<Item ID="13">
+				<Name>Queued Message Count</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Integer</Type>
+				<RangeEnumeration></RangeEnumeration>
+				<Units>count</Units>
+				<Description><![CDATA[Queued Message count. A non-zero QmsgCnt indicates that there are messages waiting to be delivered, and it can be useful for assessing the network's communication efficiency and load.]]></Description>
```

Comment:
```suggestion
				<Description><![CDATA[Queued Message count. A non-zero Queued Message Count indicates that there are messages waiting to be delivered, and it can be useful for assessing the network's communication efficiency and load.]]></Description>
```